### DISCUSSION TITLE
LTDC library not found?

### DISCUSSION THREAD

**Discussion by epikao** [stm32duino/Arduino_Core_STM32#1530] [Libraries] [answered]
Hello, is the LTDC library not supported ? I cannot find this library in the library folder?

#include "LTDC_F746_Discovery.h" shows an error, not found
I'm using platformio, stm32duino core and arduino framework

here: https://github.com/fpistm/Arduino_Core_STM32/tree/LTDC_TS_TEST_1.9.0/libraries , I can find this library - I think is not the official core?
How can I now include this library with platformio ? 

Thank you

**Comment by fpistm** [comment, marked as answer]
Hello @epikao 
this is a branch of my fork. This is only for test purpose but not officially maintained nor released.
If you want to try it, you can simply copy the library.
